### PR TITLE
Modus Data Table component with Tanstack Integration enabled with Sorting

### LINKED ISSUE #1113: Modus Data Table - Accessible column sorting
More details at https://docs.google.com/document/d/1CKKDkgKHHktu_6yxg-dF7iJ5bHAZkydo69qJMQO_2so/edit

### PR DESCRIPTION
**Pull request by msankaran0712**
Fixes #1113 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

Storybook page

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (including CHANGELOG updates)
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/src/components/modus-switch/modus-switch.scss [hunk 1]
```diff
@@ -22,6 +22,8 @@
       cursor: pointer;
       height: $rem-14px;
       position: absolute;
+      right: 0;
```
Comment (msankaran0712) [inline stencil-workspace/src/components/modus-switch/modus-switch.scss:25]: @Ankur-Koul Please revert this change and squash the commit

**Review by jeffdoolittle — APPROVED** [review]
I was able to view the progress running storyboard locally and it looks good.
